internal/router/router.go

```diff
@@ -33,24 +33,30 @@ type SliceRouterConnCtx struct {
 	LocalNsmGwPeerIP       string
 }
 
-func GetClientConnectionInfo(ctx context.Context, addr string) ([]kubeslicev1beta1.AppPod, error) {
+type workerClientConfig struct {
+	client sidecar.SliceRouterSidecarServiceClient
```

Comment:
same comment as above, naming is confusing

Reply:
Done